src/Mailer/Mailer.php

```diff
@@ -322,22 +340,75 @@ public function send(string $action, array $args = [], array $headers = []): arr
     }
 
     /**
-     * Send email directly without using a mailer action.
+     * Render content and set message body.
      *
      * @param string $content Content.
-     * @return array{headers: string, message: string}
+     * @return $this
      */
-    public function deliver(string $content = '')
+    public function render(string $content = '')
     {
-        $this->message->setBody(
-            $this->getRenderer()->render(
-                $content,
-                $this->message->getBodyTypes()
-            )
+        $content = $this->getRenderer()->render(
+            $content,
+            $this->message->getBodyTypes()
         );
-        $transport = $this->getTransport();
 
-        return $transport->send($this->message);
+        $this->message->setBody($content);
+
+        return $this;
+    }
+
+    /**
+     * Render content and send email using configured transport.
+     *
+     * @param string $content Content.
+     * @return array{headers: string, message: string}
+     */
+    public function deliver(string $content)
+    {
+        $this->render($content);
+
+        return $this->getTransport()->send($this->message);
+    }
+
+    /**
+     * Set message body.
```

Comment:
Should mention that setting a body will disable automatic rendering of the template.

Reply:
Done